packages/dso-toolkit/src/components/card/card.models.ts

```diff
@@ -5,24 +5,17 @@ import { Selectable } from "../selectable/selectable.models.js";
 import { Toggletip } from "../toggletip/toggletip.models.js";
 import { SlideToggle } from "../slide-toggle";
 
-export const imageShapes = ["normal", "wide"] as const;
-
 export interface Card<TemplateFnReturnType> {
   label: string;
-  /** Use only with `clickable: false`. */
-  href?: string;
+  href: string;
+  active?: boolean;
   mode?: Anchor["mode"];
   selectable?: Selectable<TemplateFnReturnType>;
   content?: TemplateFnReturnType;
   interactions?: Array<Button | Label | Toggletip<TemplateFnReturnType> | SlideToggle>;
-  image?: string;
-  imageAlt?: string;
-  imageShape?: (typeof imageShapes)[number];
-  /** @deprecated use `href` */
-  clickable?: boolean;
-  dsoCardClicked?: (e: CustomEvent<DsoCardClickedEvent>) => void;
+  dsoCardClick?: (e: CustomEvent<DsoCardClickEvent>) => void;
 }
 
-export interface DsoCardClickedEvent {
+export interface DsoCardClickEvent {
```

Comment:
De prefix is hier niet nodig, namespacing wordt door ESM geregeld

Reply:
Prefix is weg